app/models/tree-component.ts

```diff
@@ -0,0 +1,22 @@
+import { IActionMapping } from "angular-tree-component/dist/angular-tree-component";
```

Comment:
This should probably not be in the models folder either. Is it used anywhere else than in the treeview component?